### PR TITLE
BXMSDOC-4381 add clarification about upgrade paths

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jira:
[BXMSDOC-4381](https://issues.jboss.org/browse/BXMSDOC-4381)

### CODE REVIEW

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 1]
```diff
@@ -6,8 +6,13 @@ Automated update tools are often provided with both patch updates and new minor
 
 You can use the same automated update tool to apply both patch updates and minor release upgrades to {PRODUCT} {PRODUCT_VERSION}. Patch updates of {PRODUCT}, such as an update from version {PRODUCT_VERSION} to {PRODUCT_VERSION}.1, include the latest security updates and bug fixes. Minor release upgrades of {PRODUCT}, such as an upgrade from version {PRODUCT_VERSION}.x to 7.4, include enhancements, security updates, and bug fixes.
 
+You can update or upgrade {PRODUCT} versions 7.2.x through 7.3.1 directly. For example, you can use the 7.3.0 upgrade tool to upgrade {PRODUCT} 7.2.x to version 7.3.0. Likewise, you can directly upgrade {PRODUCT} 7.2.1 to version 7.3.1. However, you must have {PRODUCT} 7.3.1 to upgrade to version 7.4.0.
```
Comment (cimbalek) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]: Likewise, you can directly upgrade {PRODUCT} 7.2.x to version 7.3.1. However, you must have {PRODUCT} 7.3.1 to upgrade to version 7.4.0.

**Review by cimbalek — CHANGES_REQUESTED** [review]

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 2]
```diff
@@ -18,10 +23,10 @@ NOTE: Only updates for {PRODUCT} are included in {PRODUCT} update tools. Updates
 Example:
 
 * *Product:* {PRODUCT_SHORT}
-* *Version:* {PRODUCT_VERSION}.1
+* *Version:* {PRODUCT_VERSION}.0
 --
 +
-If you are upgrading to a new minor release of {PRODUCT}, such as an upgrade from version {PRODUCT_VERSION}.x to 7.4, first apply the latest patch update to your current version of {PRODUCT} and then follow this procedure again to upgrade to the new minor release.
+If you are upgrading to {PRODUCT} 7.4.x from 7.2.0 or higher, first apply the 7.3.1 patch update to your current version of {PRODUCT} and then follow this procedure again to upgrade to 7.4.x.
```
Comment (mramendi) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]: This does not sound right. If one has 7.2.0 then 7.3.1 is not a "patch update"?

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc [hunk 3]
```diff
@@ -6,8 +6,13 @@ Automated update tools are often provided with both patch updates and new minor
...
 
+You can update or upgrade {PRODUCT} versions 7.2.x through 7.3.1 directly. For example, you can use the 7.3.0 upgrade tool to upgrade {PRODUCT} 7.2.x to version 7.3.0. Likewise, you can directly upgrade {PRODUCT} 7.2.0 to version 7.3.1. However, you must have {PRODUCT} 7.3.1 to upgrade to version 7.4.0.
+
 NOTE: Only updates for {PRODUCT} are included in {PRODUCT} update tools. Updates to {EAP} must be applied using {EAP} patch distributions. For more information about {EAP} patching, see the {EAP} https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html/patching_and_upgrading_guide/[_Patching and upgrading guide_].
 
+Do not attempt to update EE7 to EE8. It is not supported but the update tool will not prevent you from doing it. If you are updating {PRODUCT} 7.3.0 to 7.3.1, update EE7 only. If you are updating {PRODUCT} 7.3.1 to 7.4.x, update EE8 only.
```
Comment (mramendi) [inline doc-content/enterprise-only/installation/patches-applying-proc.adoc]: Does every reader know what a EE7 or EE8 is? And is no EE involved if one is updating from 7.2.x?

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc (hunk 2 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc]: Yeah, it is right.

Comment (emmurphy1) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc]: @mcimbalek please confirm that the sentence that Misha is questioning is correct.

File: doc-content/enterprise-only/installation/patches-applying-proc.adoc (hunk 3 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc]: Good point, @mramendi .

Comment (emmurphy1) [reply doc-content/enterprise-only/installation/patches-applying-proc.adoc]: Updated and verified with QE.

**Pull request merged**